Resources/Private/Partials/Lists/Pagination.html

```diff
@@ -37,15 +40,28 @@
                 </f:case>
             </f:comment>
             <f:defaultCase>
-                <li class="{f:if(condition: '{page.label} == {paginator.currentPageNumber}', then:'current')}">
-                    <f:link.action action="{action}" addQueryString="true" argumentsToBeExcludedFromQueryString="{0: 'tx_dlf[page]'}" additionalParams="{'tx_dlf[page]': page.startRecordNumber}" arguments="{searchParameter: lastSearch}">{page.label}</f:link.action>
-                </li>
+                <f:switch expression="{page.nIndex}">
+                    <f:comment>add the dots</f:comment>
+                    <f:case value="...">
+                        <li class="dots">
+                            <span class="dots">...</span>
+                        </li>
+                    </f:case>
+                    <f:defaultCase>
+                        <li class="{f:if(condition: '{page.label} == {paginator.currentPageNumber}', then:'current')}">
+                            <f:link.action action="{action}" addQueryString="true" argumentsToBeExcludedFromQueryString="{0: 'tx_dlf[page]'}" additionalParams="{'tx_dlf[page]': page.startRecordNumber}" arguments="{searchParameter: lastSearch}">{page.label}</f:link.action>
+                        </li>
+                    </f:defaultCase>
+                </f:switch>
             </f:defaultCase>
         </f:switch>
     </f:for>
-    <f:if condition="{pagination.hasMorePages}">
-        <li>…</li>
-    </f:if>
+    <f:comment>is no longer needed, is provided in pagination.pagesR</f:comment>
+    <f:comment>
+        <f:if condition="{pagination.hasMorePages}">
+            <li>…</li>
+        </f:if>
+    </f:comment>
```

Comment:
If it is no longer needed (and working), please remove it entirely.

Reply:
done